app/templates/Event/schedule-list.html.twig

```diff
@@ -50,7 +50,7 @@
                         </td>
                         <td class="agenda-cell">
                             <div class="talk-info">
-                                {% include '_common/rating.html.twig' with {'rating': talk.getAverageRating, 'style': 'vertical', 'size': 'small', 'include_rating_placeholder': true} %}
+                                {% include '_common/rating.html.twig' with {'rating': talk.getAverageRating, 'userrating' : talk.getUserRating, 'style': 'vertical', 'size': 'small', 'include_rating_placeholder': true} %}
```

Comment:
I realize this is already established and I'm not going to request you to make changes for this PR but it would make sense for us to start implementing Twig Macros for this sort of thing.